hibp.js

```diff
@@ -38,6 +46,22 @@ const HIBP = {
     }
     return foundBreaches.filter(({meta}) => meta.IsVerified && !meta.IsRetired && !meta.IsSensitive && !meta.IsSpamList);
   },
+
+  async subscribeHash(sha1) {
+    const sha1Prefix = sha1.slice(0, 6).toUpperCase();
+    const path = "/range/subscribe";
+    const options = {
+      method: "POST",
+      body: {hashPrefix: sha1Prefix},
+    }
+
+    try {
+      const response = await HIBP.req(path, options);
+    } catch (error) {
+      console.error(error);
+    }
+    return response;
```

Comment:
I should probably test this locally here... but is `response` still in scope here, or is it limited to the `try { .. }` block above? Also, what is the recovery plan if we `catch` an error above?